src/main/java/Jan.java

```diff
@@ -0,0 +1,96 @@
+import java.util.Scanner;
+
+public class Jan {
+
+    public static final int MAXIMUM_TASK = 100;
+
+    public static void executeInstruction(String command, Task[] Tasks, String commandDescription){
+        System.out.println("____________________________________________________________");
+        switch(command) {
+        case "greet":
+            System.out.println("Hello! I'm Jan\n" + " What can I do for you?");
+            break;
+        case "bye":
+            System.out.println("Bye. Hope to see you again soon!");
+            break;
+        case "list":
+            System.out.println("Here are the tasks in your list:");
+            for (int i = 0; i < Task.getTotalTask(); i++) {
+                System.out.println(Tasks[i].getTaskNum() + ". " + Tasks[i]);
+            }
+            break;
+        case "done":
+            int taskNum = Integer.parseInt(commandDescription) - 1;
+            if (taskNum < Task.getTotalTask()) {
+                System.out.println("Nice! I've marked this task as done: ");
+                Tasks[taskNum].setDone(true);
+                System.out.println(Tasks[taskNum]);
+            }else{
+                System.out.println("Jan cannot find this task");
+            }
+            break;
+        case "todo":
+            Todo todo = new Todo(commandDescription,"T");
+            System.out.println("Got it. I've added this task:\n " + todo);
+            printCurrentTaskCount();
+            Tasks[Task.getTotalTask() - 1] = todo;
+            break;
+        case "deadline":
+            String[] deadlineDetails = commandDescription.split("/by");
+            Deadline deadline = new Deadline(deadlineDetails[0],deadlineDetails[1]);
+            System.out.println("Got it. I've added this task:\n " + deadline);
+            printCurrentTaskCount();
+            Tasks[Task.getTotalTask() - 1] = deadline;
+            break;
+        case "event":
+            String[] eventDetails = commandDescription.split("/at");
+            Event event = new Event(eventDetails[0],eventDetails[1]);
+            System.out.println("Got it. I've added this task:\n " + event);
+            printCurrentTaskCount();
+            Tasks[Task.getTotalTask() - 1] = event;
+            break;
```

Comment:
Perhaps you can consider applying SLAP to the blocks of code under "todo", "deadline" and "event". For example, line 35 and lines 33-34 are not in the same level of abstraction, one is a function call whereas the other is performing a computation directly.